counsel.el

```diff
@@ -3133,6 +3133,11 @@ A is the left hand side, B the right hand side."
   :group 'ivy
   :type 'string)
 
+(defcustom counsel-yank-pop-ivy-height 5
```

Comment:
Maybe `counsel-yank-pop-height` would be clear enough?

Reply:
Changed.